File: Java/LeetCode/Arrays/Easy/268-missing-number/missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        int n= nums.length;
        int sum = n*(n+1)>>1;
        int ans=0;
        for(int i=0;i<n;i++){
            ans+=nums[i];
        }
        return sum -ans;
    }
}

File: Java/LeetCode/Arrays/Easy/268-missing-number/missing-number.py
class Solution(object):
    def missingNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        n = len(nums)
        sum = (n*(n+1)>>1)
        ans = 0
        for num in nums:
            ans = ans+num
        return sum-ans